app/src/main/java/ch/epfl/sdp/peakar/database/Database.java

```diff
@@ -31,8 +32,11 @@
     public static final String CHILD_ATTRIBUTE_PEAK_LATITUDE = "latitude";
     public static final String CHILD_ATTRIBUTE_PEAK_LONGITUDE = "longitude";
     public static final String CHILD_ATTRIBUTE_PEAK_ALTITUDE = "altitude";
+    public static final String CHILD_CHALLENGES = "challenges";
+    public static final String CHILD_CHALLENGE_GOAL = "goal";
+    public final static String VALUE_POINTS_CHALLENGE = "points_challenge";
     // This is a reference to the database root
-    public static final DatabaseReference refRoot = FirebaseDatabase.getInstance("https://peakar-default-rtdb.europe-west1.firebasedatabase.app/").getReference();
+    public static final DatabaseReference refRoot = FirebaseDatabase.getInstance(dbAddress).getReference();
```

Comment:
```suggestion
    public static final DatabaseReference refRoot = FirebaseDatabase.getInstance(DATABASE_ADDRES).getReference();
```
If you did change above.

Reply:
Yes, thanks! I changed it from Android studio directly.